[2021-08-07 15:03:37,854] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: standardized_pipeline.create_standardized_table_partsupp 2021-08-07T14:55:00+00:00 [queued]>
[2021-08-07 15:03:37,896] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: standardized_pipeline.create_standardized_table_partsupp 2021-08-07T14:55:00+00:00 [queued]>
[2021-08-07 15:03:37,897] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-07 15:03:37,898] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-07 15:03:37,898] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-07 15:03:37,917] {taskinstance.py:1107} INFO - Executing <Task(MySqlOperator): create_standardized_table_partsupp> on 2021-08-07T14:55:00+00:00
[2021-08-07 15:03:37,924] {standard_task_runner.py:52} INFO - Started process 8521 to run task
[2021-08-07 15:03:37,937] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'standardized_pipeline', 'create_standardized_table_partsupp', '2021-08-07T14:55:00+00:00', '--job-id', '112', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/standardized_pipeline.py', '--cfg-path', '/tmp/tmp63p543jf', '--error-file', '/tmp/tmpbx5e2vmm']
[2021-08-07 15:03:37,939] {standard_task_runner.py:77} INFO - Job 112: Subtask create_standardized_table_partsupp
[2021-08-07 15:03:38,078] {logging_mixin.py:104} INFO - Running <TaskInstance: standardized_pipeline.create_standardized_table_partsupp 2021-08-07T14:55:00+00:00 [running]> on host ab70d6dafbc6
[2021-08-07 15:03:38,236] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=standardized_pipeline
AIRFLOW_CTX_TASK_ID=create_standardized_table_partsupp
AIRFLOW_CTX_EXECUTION_DATE=2021-08-07T14:55:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-07T14:55:00+00:00
[2021-08-07 15:03:38,239] {mysql.py:70} INFO - Executing: CREATE TABLE IF NOT EXISTS standardized.PARTSUPP (
  `PART_KEY` int NOT NULL,
  `SUPPLIER_KEY` int NOT NULL,
  `PARTSUPP_AVAILQTY` int NOT NULL,
  `PARTSUPP_SUPPLYCOST` int NOT NULL,
  `PARTSUPP_COMMENT` text NOT NULL,
  `INGESTION_TIMESTAMP` datetime DEFAULT CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
[2021-08-07 15:03:38,275] {base.py:78} INFO - Using connection to: id: dwh_id. Host: dwh, Port: 3306, Schema: , Login: root, Password: ***, extra: {}
[2021-08-07 15:03:38,361] {dbapi.py:204} INFO - Running statement: CREATE TABLE IF NOT EXISTS standardized.PARTSUPP (
  `PART_KEY` int NOT NULL,
  `SUPPLIER_KEY` int NOT NULL,
  `PARTSUPP_AVAILQTY` int NOT NULL,
  `PARTSUPP_SUPPLYCOST` int NOT NULL,
  `PARTSUPP_COMMENT` text NOT NULL,
  `INGESTION_TIMESTAMP` datetime DEFAULT CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;, parameters: None
[2021-08-07 15:03:38,403] {dbapi.py:212} INFO - Rows affected: 0
[2021-08-07 15:03:38,454] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=standardized_pipeline, task_id=create_standardized_table_partsupp, execution_date=20210807T145500, start_date=20210807T150337, end_date=20210807T150338
[2021-08-07 15:03:38,577] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-07 15:03:38,615] {local_task_job.py:149} INFO - Task exited with return code 0
